### PR TITLE
DOC: added survey link

### PR DESCRIPTION
**Pull request by mkg33**
The added link will let the interested users access our survey directly.

### CODE REVIEW

File: www/docs.rst [hunk 1]
```diff
@@ -12,3 +12,6 @@ Documentation for core SciPy Stack projects:
 
 The :doc:`getting-started` page contains links to several good tutorials
 dealing with the SciPy stack.
+
+Take part in our `user survey <http://forms.gle/LGxx5hXzrDyChj38A>` and
```
Comment (rgommers) [inline www/docs.rst]: You need to add `_` right after `j38A>` to turn this into a link.

**Comment by mkg33** [comment]
Thanks for the review and sorry for the silly typo with the link! Sure, I'm correcting it right now and will add it to news.

**Comment by rgommers** [comment]
that looks good. 

> can you please remove the `.DS_Store` file

you missed this one:)

**Comment by mkg33** [comment]
But I did leave it out when uploading files from GitHub Desktop :D Isn't it enough to uncheck the file? I'll try again...

**Comment by rgommers** [comment]
No experience with GitHub Desktop, but I think not. If you look at the diff, it's still there:

image

Now that you have it checked in, you need to explicitly remove it again (`git rm .DS_Store` on the command line).

**Comment by mkg33** [comment]
You're right, it was still there :-) Desktop is probably not the best choice for removing files..

**Comment by rgommers** [comment]
Merged, thanks @mkg33!

**Comment by rgommers** [comment]
Live now:)

**Pull request merged**